app/models/manageiq/providers/openstack/infra_manager/refresh_parser.rb

```diff
@@ -365,6 +408,16 @@ def process_collection(collection, key)
           @data_index.store_path(key, uid, new_result)
         end
       end
+
+      def process_collection_item(item, key)
```

Comment:
This method is identical to the one for [openstack cloud refresh_parser](https://github.com/ManageIQ/manageiq/blob/717ba420db95a7fdacf2d6d2d5bba88fcc477ab8/app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb#L181).  Seems like this could at least be pulled into more [common methods](https://github.com/ManageIQ/manageiq/tree/717ba420db95a7fdacf2d6d2d5bba88fcc477ab8/app/models/manageiq/providers/openstack/refresh_parser_common).